### PR TITLE
17 vs code

### LINKED ISSUE #17: Get set up with VS Code and write up
VS Code is our suggested editor to code in. So get setup in that and document the process so that other developers who choose to work on this project can do the same.

### PR DESCRIPTION
**Pull request by chloemurrell**
I have update the readme with the write up about downloading and setting up VS code. 

Closes #17

### CODE REVIEW

File: README.md
Comment (jwestw) [inline README.md]: May as well put the actual URL in there for now. Note that it will probably change in the future.

File: README.md
Comment (jwestw) [inline README.md]: Python usually starts with upper case "P"

File: README.md
Comment (jwestw) [inline README.md]: On my ONS computer these links are not working. Can we check it on your computer please before merging?

Comment (jwestw) [reply README.md]: This includes all links to .mov files.

File: README.md
Comment (jwestw) [inline README.md]: Here, `python` is a command, so should be lower case.

File: README.md
Comment (jwestw) [inline README.md]: "You can download VS Code from the [VS Code download page](https://code.visualstudio.com/download). Select the correct installer for your OS and follow the instructions."

**Review by jwestw — DISMISSED** [review]

**Comment by jwestw** [comment]
I cannot see the .mov files so consider one of the following:
- converting to gif
- converting to webm
- just doing screenshots and hosting the png file in the README

Also please speed up the video if you can.

**Comment by jwestw** [comment]
I am still not able to see the movies

image


This is on my ONS computer. 

I wonder if it's something to do with active content being blocked.

**Review by james-westwood — APPROVED** [review]
Chloe and I have discussed the videos. They are not auto playing, but when they were they were running really slowly. So not autoplaying is an acceptable compromise. 

We will split the "For Developers" section out later, if/when we have a website. For now it is all on one page.

**Review by james-westwood — APPROVED** [review]

**Review by james-westwood — APPROVED** [review]
Approving after rebase

**Pull request merged**
